src/udx.c

```diff
@@ -557,7 +557,7 @@ trigger_send_callback (udx_t *socket, udx_packet_t *pkt) {
     udx_stream_t *stream = pkt->ctx;
 
     stream->status |= UDX_STREAM_DESTROYED;
-    close_maybe(stream, UDX_ERROR_DESTROYED);
+    close_maybe(stream, 0);
```

Comment:
@mafintosh I wasn't entirely sure about this one. The JavaScript side didn't treat `UDX_ERROR_DESTROYED` as an error.

Reply:
Yea I think that's correct. It needs an error here, and unsure why I didn't before. There is still a happy path right for onclose somewhere? (ie something calls close_maybe(stream, 0) when everything is great)

Reply:
That happens here on `master`, yes:

https://github.com/mafintosh/libudx/blob/615860d0c0cf1f6102d07b36364ac1f91434d9b7/src/udx.c#L500
https://github.com/mafintosh/libudx/blob/615860d0c0cf1f6102d07b36364ac1f91434d9b7/src/udx.c#L515

Reply:
Let's try it! I'm pretty sure your fix is correct.